src/cdk-experimental/menu/context-menu.ts

```diff
@@ -108,6 +110,10 @@ export class CdkContextMenuTrigger implements OnDestroy {
     return this._menuPanel;
   }
   set menuPanel(panel: CdkMenuPanel) {
+    // If the provided panel already has a stack, that means it already has a trigger configured
+    if (panel._menuStack && isDevMode()) {
```

Comment:
I don't think it makes a difference, but conventionally the `isDevMode` check goes first